tcms/rpc/api/attachment.py

```diff
@@ -20,8 +20,8 @@ def remove_attachment(attachment_id, **kwargs):
 
         :param attachment_id: PK of attachment to remove
         :type attachment_id: int
-        :return: None
-        :raises: Exception if attachment doesn't exist,
+        :param kwargs: Dict providing the current HTTP request
```

Comment:
Correct but it provides more than the current HTTP request, see https://django-modern-rpc.readthedocs.io/en/latest/configuration_guide/methods_registration.html#access-request-protocol-and-other-info-from-a-rpc-method

Reply:
No :type: definition for kwargs ?

Reply:
Linter doesn't bring up any errors.